### PR TITLE
Implemented Help Command and Pagination

### PR DESCRIPTION
**Pull request by iShauny**
The current available pagination modules are outdated, so a local copy of the project has been imported and updated to work with discord.net v2.2.0. A basic help command has been implemented that will be refactored and improved in future PRs.

### CODE REVIEW

File: Floofbot/Floofbot.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Discord.Addons.Interactive" Version="1.0.1" />
+    <ProjectReference Include="..\Discord.Addons.Interactive\Discord.Addons.Interactive.csproj" />
```
Comment (JasperGray) [inline Floofbot/Floofbot.csproj:10]: Change to / for Linux pls. Pretty sure Windows supports forward slashes for these paths, too.

File: Floofbot/Modules/Utilities.cs [hunk 2]
```diff
@@ -16,7 +16,8 @@ public async Task Ping()
             sw.Stop();
             await msg.DeleteAsync();
 
-            EmbedBuilder builder = new EmbedBuilder() {
+            EmbedBuilder builder = new EmbedBuilder()
```
Comment (JasperGray) [inline Floofbot/Modules/Utilities.cs:19]: This file seems to only have formatting changes. Maybe include it in a separate PR or a separate commit so that it's easier to undo independently.
I'd also prefer if the newline was kept at the end of the file

File: Floofbot/Modules/Help.cs [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Floofbot.Modules
+{
+    public class Help : InteractiveBase
```
Comment (JasperGray) [inline Floofbot/Modules/Help.cs:12]: The help module and command can also get summaries

File: Floofbot/Floofbot.csproj (hunk 1 above)
Comment (iShauny) [reply Floofbot/Floofbot.csproj:10]: done

File: Floofbot/Modules/Utilities.cs (hunk 2 above)
Comment (iShauny) [reply Floofbot/Modules/Utilities.cs:19]: reverted, was autoformat by visual studio

File: Floofbot/Modules/Help.cs (hunk 3 above)
Comment (iShauny) [reply Floofbot/Modules/Help.cs:12]: This will be addressed in a future PR
